iosApp/iosApp/ComponentViews/RealtimeIndicator.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  RealtimeIndicator.swift
+//  iosApp
+//
+//  Created by esimon on 10/18/24.
+//  Copyright © 2024 MBTA. All rights reserved.
+//
+
+import SwiftUI
+
+struct RealtimeIndicator: View {
```

Comment:
suggestion (non-blocking): Realtime Indicator to me sounds more like the icon itself rather than a view also containing the prediction. Maybe something like `WithRealtimeIndicator`?